lib/sensors-store.js

```diff
@@ -38,26 +38,55 @@ function Sensor (store, logT) {
         me.data = [];
     }
 }
+
+Sensor.prototype.refresh = function() {
+    var deviceConfig = common.getDeviceConfig();
+    if (deviceConfig) {
+        if(deviceConfig) {
```

Comment:
I know I'm one to talk... but can we keep spacing and indents etc... consistent.
```suggestion
        if (deviceConfig) {
```